### PR TITLE
Feat home compass

### PR DESCRIPTION
**Pull request by mixcas**
Copies compass to home to merge functionality. Original compass still in `pages/compass/`. Maybe get rid of it later, it could be useful for debugging cuz it has useful data on view.

This also adds a little bleep mark that blinks faster as u approach destiny.

Fixes a wrong path declared on `Game.nextMinigame()`.

### CODE REVIEW

File: cordova/www/js/src/compass.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    var compensationAngle = _this.getAngle( _this.reference, _this.position, _this.destiny);
+
+    // If destiny is at West of origin
+    if( _this.position.lng > _this.destiny.lng ) {
+      compensationAngle = 360 - compensationAngle;
+    }
+
+    var angle = compensationAngle + northOrientation;
+
+    $('#compass').css({
```
Comment (varffs) [inline cordova/www/js/src/compass.js]: probably worth caching this selector

Comment (mixcas) [inline cordova/www/js/src/compass.js]: True
ill do it

File: cordova/www/home.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Home = {
+  $playGame: null,
```
Comment (varffs) [inline cordova/www/home.js]: This is not inserted but in the DOM from pageload. @mixcas why do you do this and then set the value with init()?

**Comment by varffs** [comment]
We still need to work out the correct pattern for this though. I think clicking to load the compass for next game is weird and it should just be on any home pageload. But this probably depends on how the homescreen is designed I dunno? @interglobalvision/owners

**Pull request merged**